homeassistant/components/aqvify/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""The Aqvify integration."""
+from __future__ import annotations
+
+from aqvify import AqvifyAPI, DeviceDataAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Aqvify from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    api = AqvifyAPI(entry.data[CONF_HOST], entry.data[CONF_API_KEY])
+    hass.data[DOMAIN][entry.entry_id] = DeviceDataAPI(api)
```

Comment:
You can try a fetch here an raise a config entry not ready error.

Reply:
I'm not sure that applies. This only setup the client instance for later use.